controller-server/src/test/java/com/yahoo/vespa/hosted/controller/tls/SecretStoreMock.java

```diff
@@ -0,0 +1,40 @@
+// Copyright 2018 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.vespa.hosted.controller.tls;
+
+import com.google.inject.Inject;
+import com.yahoo.vespa.hosted.controller.tls.config.TlsConfig;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+/**
+ * A secret store mock that's pre-populated with a certificate and key.
+ *
+ * @author mpolden
+ */
+@SuppressWarnings("unused") // Injected
+public class SecretStoreMock extends com.yahoo.vespa.hosted.controller.integration.SecretStoreMock {
+
+    @Inject
+    public SecretStoreMock(TlsConfig config) {
+        addKeyPair(config);
+    }
+
+    private void addKeyPair(TlsConfig config) {
+        setSecret(config.privateKeySecret(), readFile("test.key"));
```

Comment:
Try to use the test methods to create private key and certificate so we don't need to keep actual private keys in the repo

Reply:
Fixed.